index.md

```diff
@@ -9,16 +9,49 @@ tagline: The Fyne tookit for graphical applications makes it fun and easy to bui
     <div class="container">
         <div class="row">
             <div class="col-lg-8 col-lg-offset-2 text-center">
-                <h2 class="section-heading">Build the future with us!</h2>
+                <h2 class="section-heading">Fyne: the only user interface toolkit you'll ever need</h2>
                 <hr class="light">
-                <p>The Fyne toolkit provides the easiest way to develop beautiful, lightweight and user friendly native applications for desktop, mobile and beyond.
+                <p>The Fyne toolkit provides the easiest way to develop beautiful, lightweight and user friendly native applications for desktop, mobile and beyond. Fyne combines the power and ease of the Go programming language with an easy to use, high-performance library of widgets, layouts, and more.
                 Driven by design and usability principles we are breaking free of the current desktop app drudgery to create a joyous experience.</p>
-                <p>The collection of apps built using Fyne is growing all the time and many of them are open source as well! You can <a href="https://apps.fyne.io">browse the apps</a> online.</p>
                 <p>&nbsp;</p>
             </div>
         </div>
     </div>
+
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-8 col-lg-offset-2 text-center">
+                <h2 class="section-heading">What Fyne is capable of...</h2>
```

Comment:
It’s maybe a subtle thing but I’m not a fan of this heading - it implies that it’s capabilities are just graphical, and are all displayed below. I’m not sure if a header is needed here, but if it is could it be more like hinting that it’s just a small part of what’s there?

Reply:
I changed it to "A taste of Fyne...". We can also just drop that header if you want, I think it would work that way as well.